ebda_integration/ebda_integration/utils.py

```diff
@@ -64,7 +65,7 @@ def get_answers_from(survays: list = []) -> list:
 def update_answer_key_value(dictionary):
```

Comment:
"dictionary" 👀 is a data type ..

Reply:
@meaziz yes :laughing: i will improve it using `update_answer_key_value(new_param_name : dict)`